lib/src/test/java/com/auth0/jwt/impl/PayloadDeserializerTest.java

```diff
@@ -198,13 +198,15 @@ public void shouldGetNullDateWhenParsingNull() throws Exception {
     }
 
     @Test
-    public void shouldGetNullDateWhenParsingNonNumericNode() throws Exception {
+    public void shouldThrowWhenParsingNonNumericNode() throws Exception {
+        exception.expect(JWTDecodeException.class);
+        exception.expectMessage("The claim 'key' contained an unexpected value.");
+
         Map<String, JsonNode> tree = new HashMap<>();
         TextNode node = new TextNode("123456789");
         tree.put("key", node);
 
-        Date date = deserializer.getDateFromSeconds(tree, "key");
-        assertThat(date, is(nullValue()));
+        deserializer.getDateFromSeconds(tree, "key");
```

Comment:
So no assertions as now caught by the exception.expect? if `getDateFromSeconds` explodes?

Reply:
Exactly. Because the ExpectedException is "none" for every method but for those in which we override it, when we run tests they expect (or don't) to fail with X exception and Y message.